Index: .browserslistrc
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.browserslistrc b/.browserslistrc
--- a/.browserslistrc	(revision 19bac0866c539d4206ef954cdb9734518815917a)
+++ b/.browserslistrc	(date 1648717602470)
@@ -4,4 +4,3 @@
 last 2 Safari major versions
 last 2 iOS major versions
 Firefox ESR
-IE 11
